examples/examples-cpu/README.md

```diff
@@ -32,8 +34,31 @@ There are examples for GPU-accelerated ML in the `examples-gpu` project.
 
 ## Prefect
 
+This example is in the [`prefect`](prefect) folder.
+
 *You will learn:*
 - How to set up a Prefect flow for scheduled scoring of a statistical model
 - How to use a Dask cluster to distribute work
 - How to deploy the flow and run it on a schedule
 - How to deploy a flow using Prefect Cloud
+
+## Snowflake
+
+This example is in the [`snowflake`](snowflake) folder.
+
+*You will learn:*
+- How to query Snowflake and load data into a Pandas dataframe
+- How to query Snowflake and load data into a Dask dataframe
+- How to write to Snowflake with Pandas and Dask
+
+## NYC Taxi using Snowflake
+
+This example is in the [`nyc-taxi-snowflake`](nyc-taxi-snowflake) folder.
+
+The same end-to-end data science pipeline in the "NYC Taxi" example, except using [Snowflake](https://www.snowflake.com/) for data ingest and exploratory analysis. The ML examples then pull data from a Snowflake database rather than from S3.
```

Comment:
```suggestion
The same end-to-end data science pipeline in the "NYC Taxi" example, except using [Snowflake](https://www.snowflake.com/) for data ingest and exploratory analysis. The ML examples pull data from a Snowflake database rather than from S3.
```